### ISSUE TITLE
Remove curly brackets from regular strings

### ISSUE THREAD

**Issue by lachlancollins** [ota-meshi/eslint-plugin-svelte#336]
### Motivation

I've previously used the [react/jsx-curly-brace-presence](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md) rule, and I was wondering if something similar was available or could be made for this plugin.

I went through and manually removed curly brackets from one of my own projects, and got a new svelte a11y error, because it wasn't reading the curly bracket attributes correctly.

`<main role={"main"}>` -> No error
`<main role="main">` -> A11y: Redundant role 'main' svelte(a11y-no-redundant-roles)

### Description

The rule should scan for attributes wrapped in curly braces which are strings defined with `'` or `"`, and offer to remove them. It might also be worth adding a setting to force add curly braces (e.g. "always"), but since this allows your code to ignore a11y errors I'm not sure it would be recommended.

### Examples

```svelte
<!-- ✓ GOOD -->
<Foo foo="bar" />
<Foo foo='bar' />

<!-- ✗ BAD -->
<Foo foo={"bar"} />
<Foo foo={'bar'} />
```

**Comment by ota-meshi** [comment]
Thank you for posting the issue.

However, I think that rule already exists.
https://ota-meshi.github.io/eslint-plugin-svelte/rules/no-useless-mustaches/

**Comment by lachlancollins** [comment]
Whoops, I was only looking in the stylistic issues section. Thanks for the super quick reply!
